app/views/lookup_keys/_fields.html.erb

```diff
@@ -31,7 +31,7 @@
   <fieldset>
     <%= collapsing_legend _("Optional input validator"), "#optional_input_validators_#{f.object.id}", "collapsed" %>
     <div id="optional_input_validators_<%= f.object.id %>" class="collapse out">
-      <p class="help-block">
+      <p class="alert alert-info">
```

Comment:
@xprazak2 you can use the alert helper: https://github.com/theforeman/foreman/blob/develop/app/helpers/layout_helper.rb#L36
If you want to keep it looking the same:
`<%= alert :class => 'alert-info', :header => '', :text => _('Note that if you use ERB as a value of parameter, value will be validated during ENC evaluation. If value is invalid, ENC evaluation will fail.') %>` 
Or move the `Note` to the header to make it more noticeable.